### ISSUE TITLE
Unknown properties since latest version

### ISSUE THREAD

**Issue by moepkid** [cebor/angular-highcharts#137]
Hey,

I just wanted to input some options like 'height: 60%' in my Yaxis options within the chart object, but somehow it says that its not a know property. The documentation on the highcharts pages do not mention anything of the deletion of those properties. Is this related to this library?

**Comment by cebor** [comment]
Hello @moepkid, can you create a little working example, that i can have look?

**Comment by moepkid** [comment]
Will have a look! :) I think its because of the type library,

**Comment by SayakMukhopadhyay** [comment]
I think it is because of @types/highcharts . I have found lots of type inconsistensies and I have to make do with `any` most of the time.

**Comment by KyleW** [comment]
Agreed. It looks like @types/highcharts points at Highcharts 5 and many of us are using Highcharts 6. We need to publish a new version of @types/highcharts and then we can bump the reference in this project.
